src/DotNetWorker/Http/IHttpCookie.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    public interface IHttpCookie
+    {
+        string Domain { get; }
+
+        DateTimeOffset? Expires { get; }
+
+        bool? HttpOnly { get; }
+
+        double? MaxAge { get; }
+
+        string Name { get; }
+
+        string Path { get; }
+
+        SameSite SameSite { get; }
+
+        NullableBool Secure { get; }
```

Comment:
Any specific reason why this is not a `bool?`?

Reply:
No. It was supposed to be. Updating. We won't be exposing those types as part of the public surface. There's some refactoring coming that will prevent that from happening.